params/safety_config.yaml

```diff
@@ -0,0 +1,11 @@
+safety_bonds:
```

Comment:
Yes anyone who uses base hardware needs to include that first. I'm not sure why that's an issue.